### PR TITLE
Storybook Utils

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

- Moves utils from `@leafygreen-ui/lib/storybook` to a new `@lg-tools/storybook-utils`
- Updates existing story files to reference the new package
- adds `@lg-tools/storybook-utils` as a dev dependency to all packages that need it

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ We also have a link script, such that you can test components that are in develo
 The mongodb.design website will automatically import the `*.story.tsx` file from its installed package directory to render its live example. By default, the first exported story from the `*.story.tsx` file will be rendered. To specify a different story to be rendered, define the following in the Storybook file's Meta object:
 
 ```
-import { StoryMetaType } from '@leafygreen-ui/lib';
+import { StoryMetaType } from '@lg-tools/storybook-utils';
```
Comment (TheSonOfThomp) [inline DEVELOPER.md:87]: This file, and all changes in `packages/` are swapping `/lib` imports for `/storybook-utils`

File: tools/storybook-utils/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Storybook Utils
```
Comment (TheSonOfThomp) [inline tools/storybook-utils/README.md:1]: The interesting changes start here

File: tools/storybook-decorators/package.json [hunk 3]
```diff
@@ -17,14 +17,16 @@
...
     "@leafygreen-ui/tokens": "^2.5.0",
     "@lg-tools/build": "^0.3.2",
     "@storybook/react": "^7.6.17",
+    "@lg-tools/storybook-utils": "^0.0.1",
     "lodash": "4.17.21"
   },
   "peerDependencies": {
     "@leafygreen-ui/leafygreen-provider": "^3.1.11"
+  },
+  "devDependencies": {
```
Comment (shaneeza) [inline tools/storybook-decorators/package.json:30]: is this needed?

**Review by shaneeza — APPROVED** [review]

File: tools/storybook-decorators/package.json (hunk 3 above)
Comment (TheSonOfThomp) [reply tools/storybook-decorators/package.json:30]: Not anymore!

**Pull request merged**